src/tests/Equinor.ProCoSys.IPO.Domain.Tests/AggregateModels/InvitationAggregate/McPkgTests.cs

```diff
@@ -12,19 +12,20 @@ public class McPkgTests
         private McPkg _dut;
         private const string TestPlant = "PlantA";
         private const string ProjectName = "ProjectName";
+        private static readonly Guid ProjectGuid = new Guid("11111111-2222-2222-2222-333333333341");
         private const int ProjectId = 320;
         private const string CommPkgNo = "Comm1";
         private const string McPkgNo = "Mc1";
         private const string Description = "D1";
         private const string System = "1|2";
-        private readonly Project _project = new(TestPlant, ProjectName, $"Description of {ProjectName}");
+        private readonly Project _project = new(TestPlant, ProjectName, $"Description of {ProjectName}", ProjectGuid);
 
 
         [TestInitialize]
         public void Setup()
         {
             _project.SetProtectedIdForTesting(ProjectId);
-            _dut = new McPkg(TestPlant, _project, CommPkgNo, McPkgNo, Description, System);
+            _dut = new McPkg(TestPlant, _project, CommPkgNo, McPkgNo, Description, System, Guid.Empty);
```

Comment:
Add a private readonly McPkgGuid other than empty to set here, and Assert it is set in Constructor_ShouldSetProperties()